cepy/build_cephes.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2017 Gamalon, Inc.
```

Comment:
Ah, that link helps with my comment above about Cephes being proprietary software.